src/list.js

```diff
@@ -0,0 +1,19 @@
+import { getTodos } from './task.js';
+
+function displayBox() {
+  const checkBoxes = document.querySelectorAll('.check-list');
+  for (let i = 0; i < checkBoxes.length; i += 1) {
+    checkBoxes[i].addEventListener('change', (e) => {
+      const taskIndex = Number(e.target.getAttribute('data-id'));
+      let todos = JSON.parse(localStorage.getItem('todos')) || [];
+      const taskUpdate = todos.find((task) => task.index === taskIndex);
+      taskUpdate.completed = !taskUpdate.completed;
+      todos = todos.filter((task) => task.index !== taskIndex);
+      todos.push(taskUpdate);
+      localStorage.setItem('todos', JSON.stringify(todos));
+      getTodos();
+    });
+  }
+}
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] It would be better to fix the ESLint errors instead of disabling it inline. This helps us write better and clean code